test/integration/choosePage.js

```diff
@@ -65,7 +65,7 @@ describe('Choose page', () => {
     it('page options should be related to being over 25 (paid), if age question is answered 25 or older', async () => {
       const $ = cheerio.load(res.text);
 
-      expect($($('.multiple-choice .multiple--choice-option')[0]).text()).to.equal('See a sexual health professional for free near you');
+      expect($($('.multiple-choice .multiple--choice-option')[0]).text()).to.equal('See a sexual health professional near you');
```

Comment:
You can simplify this by using first, i.e.
`expect($('.multiple-choice .multiple--choice-option').first().text()).to.equal('See a sexual health professional near you');`